db/queries/items.js

```diff
@@ -0,0 +1,78 @@
+const db = require('../connection');
+
+/* Fetch Items related to a specific category */
+
+const getItem = (categoryId) => {
+  const query = `SELECT *
+  FROM items
```

Comment:
We would also need to `JOIN` `users` since the database will otherwise return all items of a particular category from all users.